### ISSUE TITLE
default CSE6SSSSSSM  has a problem

### ISSUE THREAD

**Issue by wkotlarski** [FlexibleSUSY/FlexibleSUSY#316]
Has anyone else saw this
```
Block SPINFO
     1   FlexibleSUSY
     2   2.5.0
     4   no ewsb
     5   CSE6SSM
     9   4.14.3
```
It's on the development branch.

**Comment by pathron** [comment]
No, but it can easily be that small changes shifted the point towards an ewsb failure.   I will look into it.

**Comment by wkotlarski** [comment]
You should go to sleep Peter. See you tomorrow :)

**Comment by Expander** [comment]
Thanks for looking into this, Peter. Sweet dreams (are made of this...)!

**Comment by pathron** [comment]
So I only just got back to this now...  I can reproduce the problem at least.

Unfortunately it takes a long time to build when I add cout statements to some files to see whats going on, so it may take a while....

**Comment by Expander** [comment]
I've looked at this briefly: By slightly modifying the input parameters I was not able to find a valid point. When configuring with `--enable-verbose` and running the spectrum generator I found that it tries to solve the EWSB equations numerically (not analytically). This is, because FS could not solve the EWSB equations for the desired parameters:
~~~
Writing EWSB equations to models/CSE6SSM/CSE6SSM_EWSB_equations.m
Searching for independent EWSB equations ...
Solving 5 independent EWSB equations for {m0Sq, TanTheta, vphi, ξF, LXiF0}
 
>    ...
   The EWSB solution was written to the file:
      models/CSE6SSM/CSE6SSM_semi_analytic_EWSB_solution.m
Warning: an analytic solution to the EWSB eqs. 
 
>    could not be found for the solvers: {SemiAnalyticSolver}
   An iterative algorithm will be used.  You can try to set
   the solution by hand in the model file like this:

   TreeLevelEWSBSolution = {
      { m0Sq, ... },
      { TanTheta, ... },
      { vphi, ... },
      { ξF, ... },
      { LXiF0, ... }
   };
~~~
The numerical solution algorithm does never reach the desired precision (although it comes quite close to it).

One obvious fix would be to implement the analytic solution of the EWSB equations into the FS model file like suggested by the error message above. (This is done in the `CMSSMSemiAnalytic`, for example.) One could use the expressions written to `models/CSE6SSM/CSE6SSM_EWSB_equations.m`.
